internal/reconciler/logpipeline/reconciler_test.go

```diff
@@ -89,8 +90,8 @@ func TestReconcile(t *testing.T) {
 		require.True(t, *updatedPipeline.Status.UnsupportedMode)
 	})
 
-	t.Run("should set status UnsupportedMode false if does not contains custom plugin", func(t *testing.T) {
```

Comment:
Is there a reason to remove this unit test?

Reply:
You are right, it was not intended, will revert it

Reply:
Done